src/main/java/com/minecolonies/coremod/network/messages/TransferItemsRequestMessage.java

```diff
@@ -118,38 +119,38 @@ public void messageOnServerThread(final TransferItemsRequestMessage message, fin
 
         ItemStack remainingItemStack = InventoryUtils.addItemStackToProviderWithResult(building.getTileEntity(), itemStackToTake);
 
-        if (!InventoryUtils.isItemStackEmpty(remainingItemStack))
+        if (!ItemStackUtils.isItemStackEmpty(remainingItemStack))
         {
             //If we still have some to drop, let's try the additional chests now
-            if (InventoryUtils.getItemStackSize(remainingItemStack) > 0)
+            if (ItemStackUtils.getItemStackSize(remainingItemStack) > 0)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Merge this if statement with the enclosing one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1066)